src/style.css

```diff
@@ -0,0 +1,55 @@
+.watch {
+  position: relative;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  border: 3px dotted black;
+  border-radius: 50%;
+  background-color: #fff;
+  width: 80vmin;
+  height: 80vmin;
+  margin: auto;
+}
+.watch__element {
+  position: absolute;
+  left: 50%;
+  bottom: 50%;
+  border: 1px solid black;
+  transform-origin: bottom center;
+}
+.watch__element:hover {
+  animation-play-state: paused;
+}
+.watch__arrow--big {
+  width: 12px;
+  height: 30vmin;
+  background-color: #f3f36a;
+  animation: move 60s linear forwards infinite;
+}
+.watch__arrow--small {
+  width: 12px;
+  height: 15vmin;
+  background-color: rgb(0, 166, 255);
+  animation: move 3600s linear forwards;
+}
+@keyframes move {
+  0% {
+    transform: rotate(0deg);
+  }
+  100% {
+    transform: rotate(360deg);
+  }
+}
+.watch__center {
+  position: absolute;
+  top: auto;
```

Comment:
You can remove the 'top', 'left', 'right', and 'bottom' properties as they are all set to 'auto', which is the default value.